Resources/public/css/views/dashboard-blocks.css

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-dashboardblocksview,
+.ez-view-dashboardblocksview * {
+    box-sizing: border-box;
+}
+
+.ez-view-dashboardblocksview {
+    width: 100%;
+    height: 100%;
+    padding: 2vh 6em 0;
+}
+
+.ez-view-dashboardblocksview .ez-dashboard-title {
+    margin-top: 0;
+}
+
+.ez-view-dashboardblocksview .ez-dashboard-content {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.ez-view-dashboardblocksview .ez-view-dashboardblockbaseview {
```

Comment:
the base block is not supposed to be rendered so `ez-view-dashboardblockbaseview` is not supposed to be used anywhere.